packages/core-data/src/queried-data/test/get-query-parts.js

```diff
@@ -4,14 +4,14 @@
 import { getQueryParts } from '../get-query-parts';
 
 describe( 'getQueryParts', () => {
-	it( 'parses out pagination data', () => {
+	it( 'parses out pagination data and adds to returned object', () => {
```

Comment:
Just updating these tests due to OCD. 

Given they they're checking for empty stableKeys here, I'm not very sure about this approach.